module/apmawssdkgo/sns.go

```diff
@@ -59,6 +59,8 @@ func (s *apmSNS) spanName() string { return s.name }
 
 func (s *apmSNS) resource() string { return s.resourceName }
 
+func (s *apmSNS) targetName() string { return "" }
```

Comment:
https://github.com/elastic/apm/blob/main/specs/agents/tracing-instrumentation-messaging.md#span-context-fields says this should be the queue (topic) name.

Reply:
I missed that document! Thank you for pointing it out! :bow: